management-api-server/src/main/java/com/datastax/mgmtapi/resources/NodeOpsResources.java

```diff
@@ -663,6 +663,38 @@ public Response searchIndexRebuild(
         });
   }
 
+  @POST
+  @Path("/encryption/internode/truststore/reload")
+  @Produces(MediaType.TEXT_PLAIN)
+  @ApiResponse(
+      responseCode = "200",
+      description = "Truststore reloaded successfully",
+      content =
+          @Content(
+              mediaType = MediaType.TEXT_PLAIN,
+              schema = @Schema(implementation = String.class),
+              examples = @ExampleObject(value = "OK")))
```

Comment:
:suggestion
Add the 400 response annotation for unsupported server versions
```suggestion
  @ApiResponse(
      responseCode = "200",
      description = "Truststore reloaded successfully",
      content =
          @Content(
              mediaType = MediaType.TEXT_PLAIN,
              schema = @Schema(implementation = String.class),
              examples = @ExampleObject(value = "OK")))
  @ApiResponse(
      responseCode = "400",
      description = "Unsupported Operation",
      content =
          @Content(
              mediaType = MediaType.TEXT_PLAIN,
              schema = @Schema(implementation = String.class),
              examples =
                  @ExampleObject(
                      value =
                          "Reloading the truststore manually is only possible on DSE")))
```